lib/nuorder/client.rb

```diff
@@ -68,7 +68,7 @@ def orders_with_product_details(status:)
     # Set instance variables passed in options, but fallback to module defaults
     def set_instance_variables(options)
       Nuorder::Configurable.keys.each do |key|
-        instance_variable_set(:"@#{key}", options[key] || Nuorder.instance_variable_get(:"@#{key}"))
+        instance_variable_set(:"@#{key}", options[key] || instance_variable_get(:"@#{key}"))
```

Comment:
This code no longer matches the comment. Is it necessary? I don't see any places where the instance variables will be set locally before this is called so it's misleading. Seems like this can just be `instance_variable_set(:"@#{key}", options[key])`